lua/bufferline/diagnostics.lua

```diff
@@ -114,16 +114,16 @@ local get_diagnostics = {
   end)(),
 }
 
-function M.combine(diagnosticss)
-  if #diagnosticss < 1 then return {} end
+function M.combine(diagnostics)
+  if #diagnostics < 1 then return {} end
   local result = {
-    sev_code = severity_name[diagnosticss[1].level],
+    sev_code = severity_name[diagnostics[1].level],
     errors = {},
     count = 0
   }
-  for _, diagnostics in pairs(diagnosticss) do
-    result.sev_code = math.min(result.sev_code, severity_name[diagnostics.level])
-    for severity, count in pairs(diagnostics.errors) do
+  for _, diagnostics_instance in pairs(diagnostics) do
```

Comment:
This should just use the singluar here which is
```suggestion
  for _, diagnostic in pairs(diagnostics) do
```

Reply:
Yes, but that would reuse the variable name.

Reply:
In which case something like `d` or `diag` is fine. I think it's fine to be terse here since it's to my mind pretty clear what each individual item is